core/utils/utils_test.go

```diff
@@ -19,26 +20,35 @@ import (
 
 func TestUtils_NewBytes32ID(t *testing.T) {
 	t.Parallel()
+
 	id := utils.NewBytes32ID()
 	assert.NotContains(t, id, "-")
 }
 
 func TestUtils_NewSecret(t *testing.T) {
 	t.Parallel()
+
 	tests := []struct {
 		numOfBytes int
 		wantStrLen int
 	}{
 		{12, 16}, {24, 32}, {48, 64}, {96, 128},
 	}
-	for _, tt := range tests {
-		test := tt
-		secret := utils.NewSecret(test.numOfBytes)
-		assert.Equal(t, test.wantStrLen, len(secret))
+	for _, test := range tests {
+		test := test
+
+		t.Run(fmt.Sprintf("%d_%d", test.numOfBytes, test.wantStrLen), func(t *testing.T) {
```

Comment:
nit: maybe something more descriptive here

Reply:
This is a common pattern we used here and there.
Do you suggest to add explicit test case names?

Reply:
I tend to prefer short names so that it is not cumbersome to run a particular sub-test by hand, e.g. `go test ./core/pkg -run TestSomething/complicated_sub_test_name/long_descriptions_become_messy_here`
Not a big deal though, especially when working with an IDE that assists.